bot/acl.py

```diff
@@ -107,6 +107,10 @@ def evaluate(self, user: Optional[Union[Member, User]], channel: Optional[Messag
         ) -> EvalResult:
         raise NotImplemented
 
+    @abstractmethod
+    def refers_to(self, type: str, id: int) -> bool:
```

Comment:
The control flow in the command could be greatly streamlined I think, if it were `references(self) -> Set[int]` instead. I don't think a "type" parameter is necessary, but if it is, it should be at least a `Literal[...]`